charts/incubator/bar-assistant/values.yaml

```diff
@@ -0,0 +1,47 @@
+image:
+  repository: barassistant/server
+  pullPolicy: IfNotPresent
+  tag: v3@sha256:63c9309aafdf38d6bc4a1049d2655f4db6a7d55eb0a6059daaa25e98adb4b10f
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            MEILI_MASTER_KEY: masterKey-make-it-long-for-security
+            BASE_URL: http://localhost:3010
+            MEILISEARCH_URL: ${BASE_URL}/search
+            API_URL: ${BASE_URL}/bar
+
+service:
+  main:
+    enabled: true
+    ports:
+      bar-assistant:
+        enabled: true
+        targetPort: 3000
+        port: 3000
+      meilisearch:
+        enabled: true
+        port: 7700
+      salt-rim:
+        enabled: true
+        port: 8080
```

Comment:
Might need to add "targetPorts" to them aswell. As the port can be changed the container doesnt necessarily changes its internal port. So you either need ENV to change those aswell or add the targetPort for each one of those